src/modules/uiBackendItemConverter.js

```diff
@@ -0,0 +1,41 @@
+import { TEXT, SPACER } from './itemTypes';
+
+export const spacerContent = 'SPACER_ITEM_FOR_DASHBOARD_LAYOUT_CONVENIENCE';
+export const emptyTextItemContent = 'TEXT_ITEM_WITH_NO_CONTENT';
+
+const isBackendSpacerType = item =>
+    item.type === TEXT && item.text === spacerContent;
+const isUiSpacerType = item => item.type === SPACER;
+const isTextType = item => item.type === TEXT && item.text !== spacerContent;
+
+export const convertUiItemsToBackend = items =>
+    items.map(item => {
+        let text = null;
+        if (isUiSpacerType(item)) {
+            text = spacerContent;
+        } else if (isTextType(item)) {
+            text = item.text || emptyTextItemContent;
```

Comment:
This would change the actual content stored in the backend and it's different from before.

Are we sure we want the dashboard/dashboardItem APIs to be only consumed by our dashboard app?

Reply:
Actually, no, there are no changes to what is stored in the backend.

Reply:
👍 nevermind my comment.